graphql/schemabuilder/pagination.go

```diff
@@ -420,11 +448,36 @@ func (sb *schemaBuilder) buildPaginatedField(typ reflect.Type, f interface{}) (*
 	return ret, nil
 }
 
-func (funcCtx *funcContext) extractPaginatedRetAndErr(nodeKey string, out []reflect.Value, args interface{}, retType graphql.Type) (interface{}, error) {
+func getEmbedIndex(typ reflect.Type) int {
+	for i := 0; i < typ.NumField(); i++ {
+		field := typ.Field(i)
+		if field.Type == reflect.TypeOf(PaginationArgs{}) {
+			return i
+		}
+	}
+	return 0
```

Comment:
this should be -1 or some error case i think

Reply:
done